pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerWrapper.java

```diff
@@ -149,10 +133,7 @@ public CompletableFuture<Set<String>> getAvailableBrokersAsync() {
     }
 
     private SimpleResourceUnit buildBrokerResourceUnit (String broker) {
-        String webServiceUrl = getBrokerWebServiceUrl(broker);
-        String brokerZnodeName = getBrokerZnodeName(broker, webServiceUrl);
-        return new SimpleResourceUnit(webServiceUrl,
-                new PulsarResourceDescription(), Map.of(ResourceUnit.PROPERTY_KEY_BROKER_ZNODE_NAME, brokerZnodeName));
+        return new SimpleResourceUnit(broker, new PulsarResourceDescription());
```

Comment:
is this backward-compatible? Will this update cause any lingering broker znodes?

Reply:
Yes, the znode logic at this location was completely unnecessary and removing the code doesn't impact how things are stored in znodes.

Reply:
It might be possible that this change should be left in a separate PR. It just hurts my eyes to see fuzzy match using contains. 😆